space/internal/components/migration/readonlyfixer/relationsfixer.go

```diff
@@ -31,10 +32,12 @@ func (Migration) Name() string {
 	return MName
 }
 
-func (Migration) Run(ctx context.Context, log logger.CtxLogger, store, _ dependencies.QueryableStore, space dependencies.SpaceWithCtx) (toMigrate, migrated int, err error) {
+func (Migration) Run(ctx context.Context, log logger.CtxLogger, store objectstore.ObjectStore, space dependencies.SpaceWithCtx) (toMigrate, migrated int, err error) {
 	spaceId := space.Id()
 
-	relations, err := listReadonlyTagAndStatusRelations(store, spaceId)
+	spaceStore := store.SpaceIndex(spaceId)
+
+	relations, err := listReadonlyTagAndStatusRelations(spaceStore, spaceId)
```

Comment:
It seems `listReadonlyTagAndStatusRelations` does not need spaceId parameter anymore